support-frontend/assets/components/socialShare/socialShare.jsx

```diff
@@ -24,29 +24,30 @@ type SocialMedia = {
   a11yHint: string,
 };
 
+type WindowFeatures = '' | 'menubar=no, toolbar=no, resizable=yes, scrollbars=yes, width=500, height=400';
```

Comment:
rather than writing out this string twice, we could make this a const and add a `windowFeatures: string` field to the `SocialMedia` type?

Reply:
then you could just pass `socialMedia[props.name]` to `onShare` and get rid of the `if` condition